src/helm/benchmark/scenarios/cleva_scenario.py

```diff
@@ -162,6 +162,12 @@ def get_prompt_setting(cls, task: str, subtask: str, version: str) -> PromptSett
             prompt_setting = PromptSetting(
                 instructions="",
             )
+        elif task == "disinformation":
```

Comment:
I think we should call it "fact_checking" because it is not a strict disinformation scenario defined in HELM.